Tests/XCUITests/ActivityStreamTest.swift

```diff
@@ -11,32 +11,21 @@ let allDefaultTopSites = ["Facebook", "YouTube", "Amazon", "Wikipedia", "Twitter
 class ActivityStreamTest: BaseTestCase {
     let TopSiteCellgroup = XCUIApplication().cells[AccessibilityIdentifiers.FirefoxHomepage.TopSites.itemCell]
 
-    let testWithDB = ["testActivityStreamPages", "testTopSites2Add"]
```

Comment:
I can't find `testActivityStreamPages()` in this file, so I removed the reference to this test.